src/server.js

```diff
@@ -226,9 +240,10 @@ cli.version(version)
         '-t, --tiles <mbtiles_path>',
         'Directory containing local mbtiles files to render'
     )
+    .option('-s, --default_styles <default_style_path>', 'Directory containing local styles files to render')
     .parse(process.argv)
 
-const { port = 8000, tiles: tilePath = null } = cli
+const { port = 8000, tiles: tilePath = null, default_styles: def_styles_path = null } = cli
```

Comment:
camelCase here too: `defaultStylesPath`

But the bigger issue is per above - are we dealing with a path containing multiple style JSON files, or a specific one?